doc-content/enterprise-only/installation/clustering-smart-router-install-proc.adoc

```diff
@@ -71,10 +69,10 @@ KieServicesConfiguration config = KieServicesFactory.newRestConfiguration("http:
 +
 In this example, `smartrouter.example.com` is the Smart Router URL, and `USERNAME` and `PASSWORD` are the log in credentials for the Smart Router configuration.
 
-[NOTE]
-====
-You must create containers directly against the `kie-server`. For example:
 
+. To create containers directly against the `kie-server`, enter a command similar to the following command:
```

Comment:
`kie-server` could be replaced by {KIE_SERVER}

Reply:
This whole note can be a bit confusing.
The point here is to illustrate interaction with Smart router. To fill example data the user needs to create a new container in Kie server. New container can be created either through Business central (for managed Kie server) or by sending an HTTP request as shown below (for unmanaged Kie server).
The `directly against` refer to sending HTTP request to unmanaged Kie server.

Reply:
Thanks @ksuta. I made another pass. Please let me know what you think.